### ISSUE TITLE
Add a reference to the migration docs in the Astro.resolve() section

### ISSUE THREAD

**Issue by alexwhitmore** [withastro/docs#190]
When going through the withastro/docs issues I came across https://github.com/withastro/docs/issues/147 where I learned that `Astro.resolve()` is being deprecated. In the API reference section where the information about `Astro.resolve()` lives, there should be a temporary note with reference to the migration guide.

Possibly something like:
### `Astro.resolve()`

> ⚠️ This feature is in the process of being **deprecated**. Please review the [migration guide](/en/migrate#deprecated-astroresolve) to upgrade your asset references to a future-proof option.

`Astro.resolve()` helps with creating URLs relative to the current Astro file, allowing you to reference files within your `src/` folder.

Astro _does not_ resolve relative links within HTML, such as images:

```html
<img src="../images/penguin.png" />
```

The above will be sent to the browser as-is and the browser will resolve it relative to the current **page**. If you want it to be resolved relative to the .astro file you are working in, use `Astro.resolve`:

```astro
<img src={Astro.resolve('../images/penguin.png')} />
```

**Comment by sarah11918** [comment]
Thanks for this, and yes, you're right and we're working on it!

It is on our radar as we are refreshing a LOT of the docs, and in some cases, I wait for larger PRs to merge before making smaller changes elsewhere because it makes everything go more smoothly. So not everything is fixed exactly when I notice it, but this is a good reminder of something that's really important to get done sooner rather than later.

Once a larger PR I'm expecting to merge very soon has happened, if you like, you're even welcome to use the above and make your own PR to contribute to the docs, if you like! If not, no worries. I will be on it.  :)

And thanks again for taking the time to point it out!

**Comment by alexwhitmore** [comment]
Hi Sarah, thank you for the insight! I would love to contribute to the docs. :)

**Comment by sarah11918** [comment]
Excellent! We LOVE all the contributors we can get! Once the PR I'm waiting on is merged (today?) I'll come back and ping you directly to say it's safe to grab main and make your PR!  @alexwhitmore

**Comment by sarah11918** [comment]
OK, @alexwhitmore , I made my merge and docs are once again a free-for all for PRs!  :)

I noticed a new PR come in overnight that seems related, so please do have a look at https://github.com/withastro/docs/pull/193 to see whether it address/overlaps or not. Two things to note:

1. I can't yet accept that PR because it was from an old branch of main (I've commented at the bottom there, asking if they would like to re-do from the current version of main). So that one won't get merged until/unless that's done. It's not a race.  :)

2. You are welcome to make this issue as a PR referencing the Migration guide and deprecation, and I *can* accept it since you'll be working off the most current version of main! And then, feel free to get involved/comment on the other PR, which I'm inferring wants to be more of a permanent description going forward.

And, caveat, I'm in the process of rewriting that page entirely! Who knows what it will eventually look like, but at least I'm happy for everyone to put in input, be contributors and with any luck, make my restructuring/rewrites easier!  ;)

**Comment by alexwhitmore** [comment]
Closing as this issue was made into a PR and merged :)
